charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -110,11 +110,6 @@ spec:
                   mountPath: /var/lib/clickhouse
               {{- end }}
 
-              {{- if .Values.clickhouse.useNodeSelector }}
-              nodeSelector:
```

Comment:
yeah looks like this should be at the pod spec context and not the container context? As per https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/